Library/Homebrew/manpages/brew.1.md

```diff
@@ -438,7 +438,8 @@ Note that these flags should only appear after a command.
     Display the file or directory used to cache <formula>.
 
   * `--cellar`:
-    Display Homebrew's Cellar path. *Default:* `/usr/local/Cellar`
+    Display Homebrew's Cellar path. *Default:* `$(brew --prefix)/Cellar`, or if
+    that directory doesn't exist, `$(brew --repository)/Cellar`.
```

Comment:
You are talking about what I would call a "vanilla" installation of Homebrew, as opposed to a non-vanilla installation. "Default" in this section of the document refers to the values, not the type of installation.